### ISSUE TITLE
Error with tmp file input

### ISSUE THREAD

**Issue by batosai** [Girilloid/libreoffice-file-converter#7]
Hello,
Thank you for this library. I am encountering an issue with images coming directly from my temp directory after an upload.

 ```
const outputBuffer = await libreOfficeFileConverter.convert({
    inputPath,
    format: 'jpeg',
    input: 'file',
    output: 'buffer',
})
```
sample inputPath : /tmp/se5n1y89pvf5p5hcy3yej7xn

```
Error: ENOENT: no such file or directory, open '/tmp/libreoffice-file-converter-24932-pz2qaaKe7a0J/.jpeg'
```

No problem with  a transformation in buffer. 

Thank's for your help

**Comment by Girilloid** [comment]
Hi, @batosai
Issue seems to be related to file name extraction from input path. Could you please share an example of input path with such issue?

**Comment by batosai** [comment]
Yes I put one in inputPath, it doesn’t even have an extension, I think it comes from there. However, this is the behavior of an uploader.

´´´tmp/se5n1y89pvf5p5hcy3yej7xn´´´

**Comment by Girilloid** [comment]
Fixed in [v3.0.2](https://github.com/Girilloid/libreoffice-file-converter/releases/tag/v3.0.2)
